harness/tests/experiment/utils.py

```diff
@@ -371,6 +371,28 @@ def make_workloads(tag: str) -> workload.Stream:
 ]
 
 
+def train_and_validate(
+    make_trial_controller_fn: Callable[[workload.Stream], det.TrialController],
+    steps: int = 2,
+) -> Tuple[Sequence[Dict[str, Any]], Sequence[Dict[str, Any]]]:
+    metrics: Dict[str, Any] = {"t": [], "v": []}
```

Comment:
please make this more readable
```suggestion
    metrics: Dict[str, Any] = {"training": [], "validation": []}
```